QLoggerWriter.cpp

```diff
@@ -150,10 +167,10 @@ void QLoggerWriter::write(QVector<QString> messages)
 
       for (const auto &message : messages)
       {
-         out << message;
+         out << message << Qt::endl;
```

Comment:
Is this Qt::endl really necessary? :thinking:

Reply:
Yes, you had a newline in your original code yourself. I just made it exclusive for file writing.